octodns/provider/dnsmadeeasy.py

```diff
@@ -284,6 +293,29 @@ def populate(self, zone, target=False, lenient=False):
                       len(zone.records) - before, exists)
         return exists
 
+    def supports(self, record):
+        # DNS Made Easy does not support empty/NULL SRV records
+        #
+        # Attempting to sync such a record would generate the following error
+        #
+        # octodns.provider.dnsmadeeasy.DnsMadeEasyClientBadRequest:
+        #      - Record value may not be a standalone dot.
+        #
+        # Skip the record and continue
+        def __extract_targets(data):
+            if 'value' in data:
+                return [data['value']['target'], ]
+            return [value['target'] for value in data['values']]
+
+        if record._type == "SRV" and "." in __extract_targets(record.data):
```

Comment:
Don't believe this local helper function is needed and it makes the code a little harder to follow:

```suggestion
        if value in data:
            targets = (data['value']['target'],)
        else:
            targets = [value['target'] for value in data['values']]

        if record._type == "SRV" and "." in targets:
```